packages/meditrak-server/src/apiV2/export/download/createDownloadLink.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import path from 'path';
+
+export const createDownloadLink = filePath => {
+  const fileName = path.basename(filePath);
+  return `http://localhost:8070/v1/export/download/${fileName}`;
```

Comment:
The port may be configured to something different than `8070`

Also I see that this is used in the export email body, I guess this won't work for users that don't run tupaia locally

Reply:
Oh this is just a massive oversight, I should have come back to this! Thanks for spotting 🙏